### PR TITLE
feat: refine compound v2 assets

### PR DESCRIPTION
**Pull request by chouandy**

### CODE REVIEW

**Review by Jeff-CCH — APPROVED** [review]
LGTM

File: src/logics/compound-v2/configs.ts [hunk 1]
```diff
@@ -1,14 +1,365 @@
...
+
+    return accumulator;
+  },
+  [
+    [] as common.Token[],
+    [] as Array<{ underlyingToken: common.Token; cToken: common.Token }>,
+    {} as Record<string, common.Token>,
+    {} as Record<string, common.Token>,
+  ]
+);
```
Comment (LuPoYi) [inline src/logics/compound-v2/configs.ts:357]: Nit: Is a good idea if we set `mainnetTokens` as `Token[]`, and put both `underlyingToken` and `cToken` to it, and then we can use it in test file like 
```
import { mainnetTokens } from './configs';

input: new common.TokenAmount(mainnetTokens.ETH, '1'),
output: new common.TokenAmount(mainnetTokens.cETH), '0'),
```

**Review by LuPoYi — APPROVED** [review]
LGTM

File: src/logics/compound-v2/configs.ts (hunk 1 above)
Comment (LuPoYi) [reply src/logics/compound-v2/configs.ts:357]: It seems I misunderstood the usage. My mistake.

**Pull request merged**